specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,45 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}/backupStorageRedundancies": {
+      "get": {
+        "operationId": "BackupStorageRedundancies_GetByLocation",
+        "x-ms-examples": {
+          "CosmosDBBackupStorageRedundanciesList": {
+            "$ref": "./examples/CosmosDBBackupStorageRedundanciesList.json"
+          }
+        },
+        "description": "Lists all the available backup redundancies in a region.",
```

Comment:
Lists all the available backup redundancy types in a region.

Reply:
Thanks updated